osf_tests/factories.py

```diff
@@ -406,7 +405,7 @@ def _create(cls, target_class, project=None, is_public=False,
         project.save()
 
         # Default registration parameters
-        schema = schema or get_default_test_schema()
+        schema = schema or getattr(draft_registration, 'registration_schema', False) or get_default_metaschema()
```

Comment:
re-order these arguments? `schema =. getattr(draft_registration, 'registration_schema', None) or schema or get_default_metaschema()`?

Alternatively, the more verbose, slightly more self-documenting option would be:
```
if draft_registration:
  schema = draft_registration.registration_schema
else:
  schema = schema or get_default_metaschema()
```
 In any case, we want to prevent conflicts from arising between a provided Draft and a provided Schema, unless we also want to explicitly error if passed a Draft with a schema other than what's passed in the `schema` argument.